docs/source/examples/CZML Tutorial.mystnb

```diff
@@ -175,6 +187,19 @@ extractor.add_orbit(molniya,
              )
 ```
 
-```{code-cell}
-extractor.packets
-```
+```python    
+  >>> extractor.packets
+  [...]
+  {
+     "id": "groundtrack0",
+     "availability": "2000-01-01T12:00:00Z/2000-01-01T23:59:35Z",
+     "position": {
+         "epoch": "2000-01-01T12:00:00Z",
+         "interpolationAlgorithm": "LAGRANGE",
+         "interpolationDegree": 5,
+         "referenceFrame": "INERTIAL",
+         "cartesian": [
+             0.0,
+             6280728.255793354,
+             -495875.9998745186, [...]
+  ```
```

Comment:
Same as above ... Has this been tested? If Yes can you share the screenshot?

Reply:
![image](https://user-images.githubusercontent.com/54778816/112280042-40c71b80-8c8d-11eb-9fe1-54ccb15ee702.png)
Sure! Correct me if I'm wrong, but mystnb uses the myst syntax, where codeblocks are supported
https://myst-parser.readthedocs.io/en/latest/using/syntax.html
![image](https://user-images.githubusercontent.com/54778816/112280822-132ea200-8c8e-11eb-91cb-589d101fc86f.png)
They're part of commonmark as well! https://commonmark.org/help/tutorial/09-code.html

Reply:
Something like https://github.com/executablebooks/MyST-NB/issues/272 would be better, when this becomes a thing

Reply:
Hmm, the RtD build didn't like this, however 🙃 
https://poliastro--1133.org.readthedocs.build/en/1133/examples/CZML%20Tutorial.html

Reply:
Yes correct, I was also trying to migrate it using the `codeblocks` but instead, we used `{code-block}` because when the sphinx parser ran the notebooks it will not generate these errors ... MyST works very differently with Sphinx for the RTD building ... I have tried it as I migrated the documentation from ReST to MyST completely

Reply:
Ok found your mistake ... make these changes through the Jupyter Notebook ... don't change it as it is like a code block ...

Reply:
The Jupytext Parser handles these things differently & that's why you are getting that weird error ... Try executing the code via Jupyter Notebook ... and confirm the same :)

Reply:
Should I try with ` ```{code-block} python`? You can pass cool directives with the YAML front-matter

Reply:
I don't think that will work either ... Making changes through Jupyter Notebook creates cells with all the required information regarding the input field, type of input field, execution, etc ... you can try it out but it will better to use the above-mentioned method coz of the encoding & stuff Jupyter does in the Notebook Environment...

Reply:
I am sure that using that method, you won't be able to generate the following type of formatting plus when you are making the changes to a notebook, it will be executed by the `nbsphinx` extension... So keep that in mind :)
![image](https://user-images.githubusercontent.com/66117751/112285235-1aff3e00-8cb0-11eb-8f9b-fbd8825e44dc.png)